schedule/ha/bv/migration/migration_offline_sles_ha.yaml

```diff
@@ -0,0 +1,72 @@
+---
+name: migration_offline_sles_ha
+description: >
+  Offline migration from installed SLES with HA extension
+
+  This test does an offline migration using the *DVD* media, with or without SCC registration.
+vars:
+  BOOTFROM: 'd'
+  BOOT_HDD_IMAGE: '1'
+  DESKTOP: 'textmode'
+  DROP_PERSISTENT_NET_RULES: '1'
```

Comment:
I think we can remove this setting from the test. See #11948

Reply:
Done, you were right.